src/main/java/com/revature/pojos/Subscription.java

```diff
@@ -1,44 +1,66 @@
 package com.revature.pojos;
 
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="SUBSCRIPTION")
 public class Subscription {
 	
-	private int subscriptionID;
+	@Id
+	@Column(name="SUBSCRIPTION_ID")
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	private int subscriptionId;
 	
-	private int userID;
+	@ManyToOne(cascade=CascadeType.ALL, fetch=FetchType.LAZY)
+	@JoinColumn(name="USER_ID")
+	private int userId;
```

Comment:
i think this needs to be - private User user;

Reply:
okk